autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>remove blank lines from cell</p><p> </p><p>favor <code>f-strings</code> over <code>.format</code>  as more readable since we're on python 3:</p><p> </p><pre class="ql-syntax" spellcheck="false">f's3://{bucket}/{prefix}/output'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
done